packages/amplify-e2e-tests/schemas/migrations_key/cant_add_more_gsi.graphql

```diff
@@ -0,0 +1,13 @@
+type Todo
+  @model
+  @key(fields: ["id", "version"])
+  @key(name: "SomeLSI", fields: ["id", "name"])
```

Comment:
LSI would fail the test as well. Can you remove the LSI from here as the intial schema does not have it